app/views/pages/_request_organisation_access.html.erb

```diff
@@ -13,4 +13,20 @@
     <%= govuk_link_to "Request access to a school",
           Schools::ChangeSchool.request_approval_url %>
   </p>
+
+  <details class="govuk-details" data-module="govuk-details">
+    <summary class="govuk-details__summary">
+      <span class="govuk-details__summary-text">
+      If you receive an error saying "You are already linked to this organisation"
```

Comment:
Does the error message use 'You are or 'You're'? It should use 'You're'

Reply:
The error says You are, but it's on DfE Sign In. We can raise with them, but probably won't be corrected for a while!

Reply:
That's fine then. We should write it as its written on the error message.